flutter-idea/src/io/flutter/run/daemon/DeviceSelection.java

```diff
@@ -46,8 +46,11 @@ DeviceSelection withDevices(@NotNull List<FlutterDevice> newDevices) {
     final Optional<FlutterDevice> selectedDevice = findById(newDevices, selectedId);
 
     // If there's no selected device, default the first ephemoral one in the list.
-    final FlutterDevice firstEphemoral =
+    FlutterDevice firstEphemoral =
```

Comment:
can we fix the spelling of ephemeral since we're changing this?

Reply:
Sure!